tests/unit/test_column_noise.py

```diff
@@ -184,9 +188,30 @@ def test_choose_wrong_option(dummy_dataset):
     pd.testing.assert_index_equal(original_empty_idx, noised_empty_idx)
 
 
-@pytest.mark.skip(reason="TODO")
-def test_copy_from_household_member():
-    pass
+def test_generate_copy_from_household_member(dummy_dataset):
+    config = get_configuration()[DATASETS.census.name][Keys.COLUMN_NOISE]["age"][
+        NOISE_TYPES.copy_from_household_member.name
+    ]
+    data = dummy_dataset[["age", "copy_age"]]
+    noised_data = NOISE_TYPES.copy_from_household_member(data, config, RANDOMNESS0, "age")
+
+    # Check for expected noise level
+    expected_noise = config[Keys.CELL_PROBABILITY]
+    original_missing_idx = data.index[data["age"] == ""]
+    eligible_for_noise_idx = data.index.difference(original_missing_idx)
+    data = data["age"]
+    actual_noise = (
+        noised_data[eligible_for_noise_idx] != data[eligible_for_noise_idx]
+    ).mean()
+    is_close_wrapper(actual_noise, expected_noise, 0.02)
+
+    # Noised values should be the same as the copy column
+    noised_mask = noised_data[eligible_for_noise_idx] != data[eligible_for_noise_idx]
+    noised_idx = noised_mask[noised_mask].index
+    assert (
+        dummy_dataset.loc[noised_idx, COPY_HOUSEHOLD_MEMBER_COLS["age"]]
+        == noised_data.loc[noised_idx]
+    ).all()
```

Comment:
I've changed the variable name from mask. It technically was a mask because it was a bool series but it is not the same length as dummy_dataset and noised_data which is why I need to find the index.